src/polytope/core/uuidgen.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+# collision probability in 150,000 entries ~ 1%
+# lowercase alphabet + digit except [l, 1, o, 0]
+DEFAULT_ALPHABET = "abcdefghijkmnpqrstuvwxyz23456789"
+DEFAULT_LENGTH = 8
+
+
+class PolytopeUUID:
+    """UUID generator class."""
+
+    def __init__(self, alphabet: str = None, length: int = 0):
+        """! PolytopeUUID class initializer.
+
+        @param alphabet     alphabet for generating uuid
+        @param length       length of uuid
+        """
+        if alphabet is None:
+            self._alphabet = DEFAULT_ALPHABET
+        else:
+            self._alphabet = alphabet
+
+        if length == 0:
+            self._length = DEFAULT_LENGTH
+        else:
+            self._length = length
```

Comment:
How about giving `DEFAULT_ALPHABET` and `DEFAULT_LENGTH` as default parameter values instead of `None` and `0`?